README.md

```diff
@@ -1,19 +1,35 @@
-# Vim [![Version](http://vsmarketplacebadge.apphb.com/version/vscodevim.vim.svg)](http://aka.ms/vscodevim) [![Build Status](https://travis-ci.org/VSCodeVim/Vim.svg?branch=master)](https://travis-ci.org/VSCodeVim/Vim) [![Slack Status](https://vscodevim-slackin.azurewebsites.net/badge.svg)](https://vscodevim-slackin.azurewebsites.net)
+<h1 align="center">
```

Comment:
Yea I get it's not ideal from purity standpoint, but we're not the first to [do this](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome).
It's supported by Github and the VSCode marketplace.